cmake/iDynTreeDependencies.cmake

```
@@ -95,6 +95,7 @@ idyntree_handle_dependency(WORHP DO_NOT_SILENTLY_SEARCH)
 # Workaround for https://github.com/robotology/idyntree/issues/599
 # NO_MODULE passed to avoid that the Findassimp of YCM is used instead, https://github.com/robotology/idyntree/pull/832
 idyntree_handle_dependency(assimp DO_NOT_SILENTLY_SEARCH NO_MODULE MAIN_TARGET assimp::assimp)
+idyntree_handle_dependency(sdformat MAIN_TARGET sdformat::sdformat)
```

```suggestion
idyntree_handle_dependency(sdformat DO_NOT_SILENTLY_SEARCH MAIN_TARGET sdformat::sdformat)
```

Can we add the `DO_NOT_SILENTLY_SEARCH` option here?

---

It should have been addressed in https://github.com/robotology/idyntree/pull/1269/commits/d944e74da53fd696d798c19e4d98464850d8cd1b, or am I missing something?